disnake/message.py

```diff
@@ -1398,6 +1398,15 @@ def system_content(self) -> Optional[str]:
         if self.type is MessageType.auto_moderation_action:
             return self.content
 
+        if self.type is MessageType.interaction_premium_upsell:
+            return self.content
+
+        if self.type is MessageType.guild_application_premium_subscription:
+            application_name = (
+                self.application["name"] if self.application else "a deleted application"
+            )
```

Comment:
```suggestion
            application_name = (
                self.application["name"]
                if self.application and "name" in self.application
                else "a deleted application"
            )
```
thoughts on this? The documentation states it's a "partial" application object, and while I don't expect the name to be missing, it would be more safe

Reply:
sounds good